azure-pipelines.yml

```diff
@@ -81,12 +83,6 @@ jobs:
       msbuild $(Build.SourcesDirectory)/Permissions_UITest/Permissions_UITest/Permissions_UITest.csproj
     displayName: Build Permissions_UITest
 
-  - script: bundle exec bin/ci/appcenter.rb
-    env:
-      APPCENTER_ACCESS_TOKEN: $(AppCenterAPITokenProd)
-      SOURCE_DIRECTORY: $(Build.SourcesDirectory)
-    displayName: Run Appcenter Test
```

Comment:
remove test job? why?

Reply:
We don't use calabash tests as a CI check anymore.